### PR TITLE
:bug: Unify the README files on kubestellar.io and on kubestellar github

### PR DESCRIPTION
**Pull request by ezrasilvera**
1. Unify the README files on kubestellar.io and on kubestellar github
2. Update contributors 

## Related issue(s)

Fixes #

### CODE REVIEW

**Review by pdettori — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -8,43 +8,34 @@
 <br/>
 
 ## Multi-cluster Configuration Management for Edge, Multi-Cloud, and Hybrid Cloud
-<br/>
 
-**NOTE**: Our documentation generally appears on the website [kubestellar.io](https://kubestellar.io), although release 0.20.0 is not documented there and release 0.21.0 is only slightly documented there. For releases 0.20.0 and later, the documentation is also made to be viewed directly on GitHub's view of the source documents; find them [there](docs/content/direct). Release `X.Y.Z` is the commit with Git tag `vX.Y.Z`.
```
Comment (MikeSpreitzer) [inline README.md:13]: Why was this line deleted? I thought it is still true. Retreating from docs being viewable directly through GitHub is a major policy change, not something that two people should do without consulting anyone else.
